ocs_ci/ocs/platform_nodes.py

```diff
@@ -111,7 +111,7 @@ def get_vms(self, nodes):
             self.cluster_name, self.datacenter, self.cluster
         )
         node_names = [node.get().get('metadata').get('name') for node in nodes]
-        return [vm for vm in vms_in_pool if vm.name in node_names]
+        return [[vm for vm in vms_in_pool if vm.name in node] for node in node_names]
```

Comment:
Won't this return a list that contains a list per node we want to return? 
i.e. `[['node1'], ['node2']]`

I think what we want is something like this:

```python
vms = []
for node in node_names:
    node_vms = [vm for vm in vms_in_pool if vm.name in node]
    vms.extend(node_vms)
return vms
```

Reply:
@clacroix12 great catch, thanks for the suggestion!